pkgs/applications/audio/miniplayer/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy39
+, colorthief
+, ffmpeg-python
+, mpd2
+, pillow
+, pixcat
+, requests
+, ueberzug
+}:
+
+buildPythonPackage rec {
+  pname = "miniplayer";
+  version = "1.7.0";
+  format = "pyproject";
+  disabled = !isPy39;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-JUlUFj/5DOTLa1XCZX/9Nj3Z9W+k4gnpJONiS4qNBIU=";
+  };
+
+  propagatedBuildInputs = [
+    colorthief
+    ffmpeg-python
+    mpd2
+    pillow
+    pixcat
+    requests
+    ueberzug
+  ];
```

Comment:
pythonImportsCheck and tests are missing.

Reply:
We should still add this for this package.

Reply:
This package doesn't expose any modules. It's basically a single file Python executable.
```
[azahi@melian:~/src/nixpkgs]$ nix build -L .#miniplayer

[azahi@melian:~/src/nixpkgs]$ tree ./result
./result
├── bin
│   └── miniplayer
├── lib
│   └── python3.9
│       └── site-packages
│           └── miniplayer-1.7.0.dist-info
│               ├── INSTALLER
│               ├── LICENSE
│               ├── METADATA
│               ├── RECORD
│               ├── REQUESTED
│               ├── WHEEL
│               ├── direct_url.json
│               └── top_level.txt
└── nix-support
    └── propagated-build-inputs

6 directories, 10 files

```

Reply:
We should add a very short comment to make sure this question does not come up again.

Reply:
Done [here](https://github.com/NixOS/nixpkgs/compare/79c2c29c3ffc1d8effca797608a774d54feafb1d..78758c07b6dd9e01bd8c9fe36ec8465cf1368f38#diff-87001ca47ad590f001ab986c1cdf7a882c756d5e024ed4d7c99b05b06effdc37R27).